examples/cluster/playground/Dockerfile

```diff
@@ -29,9 +29,10 @@ RUN pip3 install --upgrade pip==10.0.1
 RUN pip3 install --upgrade virtualenv==15.1.0
 
 ADD . /src
-ENV PYTHONPATH=/src
+ENV PYTHONPATH=/src CRYPTOGRAPHY_DONT_BUILD_RUST=1
```

Comment:
we don't need this anymore since we've pinned to a cryptography version that doesn't need rust to build wheels

Reply:
Removed. And checked the fork's Action Run and it still passes 👍